source/patterns/@aws-solutions-constructs/aws-events-rule-kinesisstream/package.json

```diff
@@ -0,0 +1,83 @@
+{
+  "name": "@aws-solutions-constructs/aws-events-rule-kinesisstream",
```

Comment:
Suggest name change of the pattern from `aws-events-rule-kinesisstream` to `aws-events-rule-kinesisstreams` to be consistent with other `aws-kinesisstreams-*` patterns in the library, it will entain changing the directory names as well, sorry should have caught it the first time..

Reply:
Done